xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -634,6 +631,28 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       }
       m_mime = "video/av01";
       m_formatname = "amc-av1";
+
+      if (m_hints.hdrType == StreamHdrType::HDR_TYPE_DOLBYVISION && m_hints.dovi.dv_profile == 10)
+      {
+        bool displaySupportsDovi{false};
+        bool mediaCodecSupportsDovi{false};
```

Comment:
You don't need to define here, declaring is enough because this can't be undefined after the next line, and the choice of initializer value here (true/false) has no effect.